theano/tensor/basic.py

```diff
@@ -447,7 +447,6 @@ def get_scalar_constant_value(orig_v, elemwise=True,
                 max_recur > 0):
             max_recur -= 1
             if isinstance(v.owner.op, (Alloc, DimShuffle, Rebroadcast,
-                                       compile.ops.OutputGuard,
```

Comment:
In case of old pickled stuff. I'm not 100% it is needed, but I prefer to be safe. It cost nothing to keep it there as we already keep the class. Maybe add a comment like:

We keep it here just in case some old pickled file need that.